Makefile

```diff
@@ -0,0 +1,41 @@
+AWS_ACCOUNT_NAME ?= michael
+AWS_DEFAULT_REGION ?= eu-west-1
+PYTHON ?= python3
+BEHAVE ?= behave
+KEYFILE ?=.anslk_random_testkey
+
+LIBFILES := $(shell find backup_cloud_ssm -name '*.py')
+
+all: lint test
+
+# pytest-mocked is much faster than non-mocked which is slower even than
+# the functional tests so run it first, then behave then ffinally the
+# full pytest tests so that failures are detected early where possible.
+test: develop pytest-mocked behave pytest 
+
+behave:
```

Comment:
Can we add the comments for the each command (ideally on the same line as the command name) - see the comment about the `help` command below.

Reply:
@michael-paddle could you please add this? It is not super clear what each command does